lib/rest_kubernetes.rb

```diff
@@ -5,8 +5,8 @@ module Rest
     module Kubernetes
       extend Helper
 
-      def self.populate(path, base_opts, opts, type)
-        populate_common("/apis/#{type}.k8s.io/<api_version>", path, base_opts, opts)
+      def self.populate(path, base_opts, opts)
+        populate_common("/api/<api_version>", path, base_opts, opts)
```

Comment:
How about
```
def self.populate(path, base_opts, opts, type: nil)
  type_str = type ? "/#{type}.k8s.io" : ""
  populate_common("/apis#{type_str}/<api_version>", path, base_opts, opts)
```

I will apply this patch now to get things fixed. But for future resolution.

Reply:
sounds good.  It should be `type_str = type ? "s/#{type}.k8s.io" : ""` though because the other format is `apis`